testsuite/integration/microprofile-tck/config/src/test/resources/arquillian-bootable.xml

```diff
@@ -9,17 +9,12 @@
             <property name="installDir">${install.dir}</property>
             <property name="jarFile">${bootable.jar}</property>
 
-            <property name="javaVmArguments">${server.jvm.args}</property>
-            <property name="jbossArguments">${jboss.args}</property>
-            <!-- -Djboss.inst is not necessarily needed, only in case the test case needs path to the instance it runs in.
-                 In the future, Arquillian should capable of injecting it into @ArquillianResource File or such. -->
+            <property name="javaVmArguments">${microprofile.jvm.args}</property>
+            <property name="managementAddress">127.0.0.1</property>
```

Comment:
Is it wise to hardcode IPv4 address here? IMHO this might not work in IPv6 only environment... (if that is our concern).

Reply:
Hi @honza-kasik - I can understand your point but I simply decided to change the contents of these configuration files (which I inherited from a previous change of mine) and make them similar to the corresponding - and already existing - arquillian configuration, see for instance https://github.com/wildfly/wildfly/blob/master/testsuite/integration/microprofile-tck/config/src/test/resources/arquillian.xml
I expect for this last one to be already tested and working but maybe I am wrong, WDYT?

Reply:
Thank you! If that is already in use, there are probably no issues with that.